Source/ROLib/Modules/ModuleROTank.cs

```diff
@@ -770,6 +770,28 @@ public void EditDimensions()
             }
         }
 
+        private bool _isRecoloringWindowOpen = false;
+
+        private void OnMouseOver()
+        {
+            if (!HighLogic.LoadedSceneIsEditor)
+            {
+                return;
+            }
+            
+            if (!Input.GetKeyDown(onHoverKeyCode)) return;
```

Comment:
onHoverKeyCode doesn't exist yet. I have added this to the new code so will test and then close this PR.